jenkins/groovy/synthetic-longlived/Jenkinsfile

```diff
@@ -0,0 +1,103 @@
+def versioningManagerSetGA(environment, release_number) {
```

Comment:
This seems like the wrong file name?

Reply:
that wasn't intended to be included in this PR :)